pkgs/applications/networking/instant-messengers/cinny-desktop/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv
+, lib
+, dpkg
+, fetchurl
+, autoPatchelfHook
+, openssl
+, gnused
+, webkitgtk
+, makeWrapper
+, disableCompositing ? false # https://bugs.webkit.org/show_bug.cgi?id=180739
+}:
+
+stdenv.mkDerivation rec {
+
+  name = "cinny-desktop";
+  version = "2.0.4";
+
+  src = fetchurl {
+    url = "https://github.com/cinnyapp/cinny-desktop/releases/download/v${version}/cinny_${version}_amd64.deb";
```

Comment:
Have you considered building it from scratch instead of repackaging a deb? It would be preferable to do so for Nixpkgs. If you still want to use debs, you must add this
```nix
sourceProvenance = with sourceTypes; [ binaryNativeCode ];
```
to `meta`.

Reply:
Tauri doesn't seem to offer the possibility to build basic binaries but only AppImage or Debian package on Linux.

Reply:
Okay then.